redaxo/src/core/lib/Fragment/Component/Button.php

```diff
@@ -0,0 +1,147 @@
+<?php
+
+namespace Redaxo\Core\Fragment\Component;
+
+use Redaxo\Core\Fragment\Slot;
+use rex_fragment;
+
+class Button extends rex_fragment
+{
+    private string $fileName = 'core/Component/Button.php';
```

Comment:
Könnte auch eine Konstante sein, oder?
Oder vielleicht sogar unten einfach direkt den Wert nutzen, da eh nur eine Stelle?

Reply:
Hab den Namen direkt in der Methode notiert 
https://github.com/redaxo/redaxo/pull/5516/commits/4b4b00980fff8ac0601e94ef49fec87cbb93a6a4